package/lib/compileFunctions.js

```diff
@@ -83,6 +89,14 @@ module.exports = {
 
         funcTemplate.properties.httpsTrigger = {};
         funcTemplate.properties.httpsTrigger.url = url;
+
+        if (funcObject.allowUnauthenticated === true) {
```

Comment:
Let's simply put it as ` if (funcObject.allowUnauthenticated)` (it's not natural in JS to confirm on `true` specifically)

Reply:
🤦 

Well thats super obvious now that you said it. Thanks for catching this (: